test/e2e/e2e_test.go

```diff
@@ -70,6 +67,8 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.configurableClusterOptions.Region, "e2e.aws-region", "us-east-1", "AWS region for clusters")
 	flag.Var(&globalOpts.configurableClusterOptions.Zone, "e2e.aws-zones", "Deprecated, use -e2e.availability-zones instead")
 	flag.Var(&globalOpts.configurableClusterOptions.Zone, "e2e.availability-zones", "Availability zones for clusters")
+	flag.StringVar(&globalOpts.configurableClusterOptions.AWSOidcS3BucketName, "e2e.aws-oidc-s3-bucket-name", "", "AWS S3 Bucket Name to setup the OIDC provider in")
```

Comment:
This should probably default with the old value for CI purposes - `hypershift-ci-oidc`

Reply:
I don't think we should default this value just for CI, since this value is definitely not usuable by anyone attempting to run this e2e test outside of CI.

Going off of this line of thinking, for the S3 OIDC bucket, I added a check so it fails fast if a value is not specified: https://github.com/openshift/hypershift/pull/2742/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R167-R169

Reply:
Ok, I'm good with this. I previously missed where it was passed in thru `hack/ci-test-e2e.sh`.